ceph_medic/checks/common.py

```diff
@@ -204,3 +204,31 @@ def check_fsid_per_daemon(host, data):
             failed = True
     if failed:
         return code, msg
+
+
+def check_multiple_running_mons(host, data):
+    code = 'ECOM10'
+    msg = 'multiple running mons found: %s'
+    sockets = data['ceph']['sockets']
+    running_mons = []
+    for socket_name in sockets.keys():
+        if "mon." in socket_name:
+            running_mons.append(socket_name)
+    if len(running_mons) > 1:
+        return code, msg % ','.join(running_mons)
+
+
+def check_colocated_running_mons_osds(host, data):
+    code = 'ECOM11'
```

Comment:
I'm thinking this should be a warning, not an error. Thoughts?

Reply:
I think you are right. I'll update